src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int max = 100;

```

Comment:
violates constant naming convention
```suggestion
    private static final int max = 100;

```

Reply:
not fixed